src/components/ViewOnly.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable react/destructuring-assignment */
```

Comment:
We talked about it already, but generally you shouldn't disable eslint rules without a good reason.